### PR TITLE
[KOGITO-5104] - [1.5.x] - RHPAM Kogito operator creates wrong default image tags

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
Many thanks for submitting your Pull Request :heart:! 

https://issues.redhat.com/browse/KOGITO-5104
Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains a link to the JIRA issue
- [ ] Pull Request contains a description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

### CODE REVIEW

File: bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -268,6 +268,16 @@ spec:
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: BUILDER_IMAGE
+                  value: rhpam-kogito-builder-rhel8
+                - name: RUNTIME_IMAGE
+                  value: rhpam-kogito-runtime-jvm-rhel8
+                - name: RUNTIME_NATIVE_IMAGE
+                  value: rhpam-kogito-runtime-native-rhel8
```
Comment (radtriste) [inline bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml:276]: we don't have native support on 1.5.x so I guess we should not document it in the manifests

```suggestion
```

File: config/manager/manager.yaml [hunk 2]
```diff
@@ -34,5 +34,15 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.annotations['olm.targetNamespaces']
+          - name: BUILDER_IMAGE
+            value: rhpam-kogito-builder-rhel8
+          - name: RUNTIME_IMAGE
+            value: rhpam-kogito-runtime-jvm-rhel8
+          - name: RUNTIME_NATIVE_IMAGE
+            value: rhpam-kogito-runtime-native-rhel8
```
Comment (radtriste) [inline config/manager/manager.yaml:42]:
```suggestion
```

File: rhpam-kogito-operator.yaml [hunk 3]
```diff
@@ -1437,6 +1437,16 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.annotations['olm.targetNamespaces']
+        - name: BUILDER_IMAGE
+          value: rhpam-kogito-builder-rhel8
+        - name: RUNTIME_IMAGE
+          value: rhpam-kogito-runtime-jvm-rhel8
+        - name: RUNTIME_NATIVE_IMAGE
+          value: rhpam-kogito-runtime-native-rhel8
```
Comment (radtriste) [inline rhpam-kogito-operator.yaml:1445]:
```suggestion
```

File: go.mod [hunk 4]
```diff
@@ -60,3 +60,5 @@ replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.2+inc
 
 // Required by Helm
 replace github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503-38ab9da00309
+
+replace github.com/kiegroup/kogito-operator => /home/vaibhavjain/RedHatRepo/kogito-operator
```
Comment (radtriste) [inline go.mod:64]: please set PR as draft until this is removed, to avoid any merge errors

**Review by radtriste — CHANGES_REQUESTED** [review]

File: bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (vaibhavjainwiz) [reply bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml]: removed

File: config/manager/manager.yaml (hunk 2 above)
Comment (vaibhavjainwiz) [reply config/manager/manager.yaml]: removed

File: rhpam-kogito-operator.yaml (hunk 3 above)
Comment (vaibhavjainwiz) [reply rhpam-kogito-operator.yaml]: removed

File: go.mod (hunk 4 above)
Comment (vaibhavjainwiz) [reply go.mod]: done

**Review by radtriste — APPROVED** [review]

**Review by spolti — APPROVED** [review]

**Review by spolti — APPROVED** [review]

**Pull request merged**